### ISSUE TITLE
ignore the prepending vX.X.X

### ISSUE THREAD

**Issue by knownasilya** [Strider-CD/ecosystem-client#1]
If a user releases a package with the name `v1.0.3` the lib can't handle that and crashes with no error.

v1.0.3 should be treated as 1.0.3

**Comment by kfatehi** [comment]
I would rather we standardize on no "v" than handle edge cases. In oyher words if we handle v1.1.1 then maybe we have to handle v.1.1.1 or version-1.1.1 or other non-semver things. Thoughts?

**Comment by knownasilya** [comment]
@keyvanfatehi yeah you have a point https://github.com/mojombo/semver/issues/204

It's not valid semver, although the `semver` module strips out leading 'v's.

**Comment by knownasilya** [comment]
So really we just need a friendly error that says the versions don't match, instead of throwing a 500 or something.
